schema/core/primitive/array_of_numbers.json

```diff
@@ -0,0 +1,9 @@
+{
+    "schemaId": "core/primitive/array-of-numbers",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "array of numbers",
+    "type": "array",
+    "items": {
+        "type": "number"
+    }
+}
```

Comment:
Nit: we have array_of_3_numbers, and could be inheriting from this...

Reply:
Ok, I'll update `array_of_3_numbers` accordingly